testbot.py

```diff
@@ -73,6 +73,14 @@ async def handle_command(message: discord.message):
         await join_game(arguments, message)
         return
 
+    if content.startswith(f'{PREFIX} poll'):
+        logging.info('Request to make a new poll')
+        user_poll = await format_poll(message)
```

Comment:
Instead of handling the validation here, why not add this `format_poll` to the start of the `make_poll` function? It will tidy up this `handle_command` handler

Reply:
Fixed. I put everthing from `make_poll` into `format_poll` to make a lot cleaner.